docker/README.md

```diff
@@ -103,30 +114,86 @@ You may also want to modify the `-p 80:80` to utilize other ports instead. Refer
 **Note**: if you're using the GPU Docker, skip ahead to the next section.
 
 ```bash
-docker run -it -p 80:80 mms_image:latest
+$ docker run -i -t -d -p 80:80 --name mms -v /home/user/models:/models mms_image:latest
+# This runs the docker image in a detached mode
 ```
-Now that you have a prompt inside the container, the final step is to run the following command:
 
+To run the MMS with $HOSTNAME as its endpoint, or for `nginx's server_name` to be configured to `$HOSTNAME` run the following command.
 ```bash
-cd mms_docker_cpu && ./launch.sh
+$ echo $HOSTNAME
+user.ant.amazon.com
+
+$ docker run -i -t -d -p 80:80 --name mms -v /home/user/models:/models -e MMS_HOST=$HOSTNAME mms_image:latest
```

Comment:
-itd

Reply:
/home/user/models - probably not a match for user's system. lets pick a nominal location like ~/models and use it throughout the doc

Reply:
I think we need to give absolute path to the model directory, well that was definitely required for mounting a file into the docker instance. So, I added the absolute path. I could change the rest of the doc from "~/models" to "/home/users/models"